samples/Tutorial/Index.fs

```diff
@@ -46,7 +46,7 @@ type Service<'t> internal (decider : Equinox.Decider<Events.Event<'t>, Fold.Stat
         decider.Query id
 
 let create<'t> resolve indexId =
-    Service(streamName indexId |> resolve)
+    Service(target indexId |> resolve)
```

Comment:
Could be `Service(target >> resolve)` for consistency

Reply:
Same as for Set